types/src/transaction/use_case.rs

```diff
@@ -52,6 +53,18 @@ impl UseCaseAwareTransaction for SignedTransaction {
                     ContractAddress(*module_id.address())
                 }
             },
+            Payload(TransactionPayloadInner::V1 {
```

Comment:
this feels better to have a self.payload().entry_function() -> Option\<EntryFunction\> to avoid duplicates

Reply:
Done.